api/hydrophoneApi.go

```diff
@@ -446,3 +470,35 @@ func (a *Api) isAuthorizedUser(tokenData *shoreline.TokenData, userId string) bo
 		return false
 	}
 }
+
+// userId is member of a Team
+func (a *Api) isTeamMember(userID string, team store.Team) (bool, error) {
+	for i := 0; i < len(team.Members); i++ {
+		if team.Members[i].UserID == userID {
```

Comment:
```suggestion
		if team.Members[i].UserID == userID && team.Members[i].InvitationStatus == "accepted" {
```
Otherwise it won't allow to resend the invite